WordPress/Classes/ViewRelated/Notifications/ReplyTextView/ReplyTextView.swift

```diff
@@ -104,6 +98,16 @@ import WordPressShared.WPStyleGuide
         textView.replace(newRange, withText: replacementText)
     }
 
+    @objc open func collapseReplyTextView() {
+        isExpanded = false
+        self.headerView.isHidden = true
+        UIView.animate(withDuration: 0.5, animations: {
+            self.expandButton.isHidden = false
+            self.replyButton.isHidden = false
+            self.refreshInterface()
+        }) { _ in
```

Comment:
We generally prefer not to use trailing closure syntax for cases like this that have multiple blocks – I think it's often  clearer and more consistent to name the last parameter. 🙂

Reply:
I agree!  In these animations I may not need the completion block so it may be removed before the PR is complete. :)